tox.ini

```
@@ -1,11 +1,10 @@
 [tox]
-envlist = py{38,311,312},pipdeptree{,-requirements},flake8,report
+envlist = py{311,312}-{pipdeptree,pipdeptree-requirements,flake8},report
 
 [gh-actions]
 python =
-    3.8: py38,pipdeptree{,-requirements},flake8
-    3.11: py311,pipdeptree{,-requirements},flake8
-    3.12: py312,pipdeptree{,-requirements},flake8
+    3.11: py311-{pipdeptree,pipdeptree-requirements,flake8}
+    3.12: py312-{pipdeptree,pipdeptree-requirements,flake8}
```

Same problem as above. Should be fixable like this:

```suggestion
    3.11: py311{,-pipdeptree,-pipdeptree-requirements,-flake8}
    3.12: py312{,-pipdeptree,-pipdeptree-requirements,-flake8}
```